README.md

```diff
@@ -22,6 +22,9 @@ The [scripts](scripts) folder contains scripts for many frequent tasks:
 
 * `build`: compiles all the project and runs all the integration tests projects that are listed in [build](scripts/build) script file
 
+Note:
+Git Bash is recommended.
```

Comment:
I would put this under the overall heading for integration tests because you have to use it for visual studio too. Also maybe say "If using Windows, Git Bash is recommended"